Sources/PointFreeRouter/Routes.swift

```diff
@@ -1,10 +1,27 @@
 import ApplicativeRouter
 import Either
+import HttpPipeline
 import Models
 import PointFreePrelude
 import Prelude
 import Stripe
 
+public enum EncryptedTag {}
+public typealias Encrypted<A> = Tagged<EncryptedTag, A>
+
+extension Tagged where Tag == EncryptedTag, RawValue == String {
+  public init?(_ text: String, with secret: String) {
+    guard
+      let string = encrypted(text: text, secret: secret)
```

Comment:
since we're going with removing all encryption stuff from the routes, we could move all of this back to `PointFree` and remove the `HttpPipeline` dependency from `PointFreeRouter`. would speed up cold builds for the router a lot.